data/mods/More_Survival_Tools/professions.json

```diff
@@ -0,0 +1,110 @@
+[
+    {
+        "type": "profession",
+        "ident": "nomad",
+        "name": "Nomad",
+        "description": "You always prefered the comfort of the open sky, far from the complexities of modern life. Though from the looks of it, things have changed since last time you've been anywhere near civilization.",
+        "points": 4,
+        "skills": [
+            {
+                "level": 4,
+                "name": "survival"
+            },
+            {
+                "level": 3,
+                "name": "fabrication"
+            },
+            {
+                "level": 2,
+                "name": "tailor"
+            },
+            {
+                "level": 3,
+                "name": "gun"
+            },
+            {
+                "level": 2,
+                "name": "archery"
+            },
+            {
+                "level": 3,
+                "name": "cooking"
+            },
+            {
+                "level": 1,
+                "name": "firstaid"
+            }
+        ],
+        "items": {
+            "both": [
+                "armor_farmor",
+                "backpack_leather",
+                "boots_fur",
+                "gloves_fur",
+                "hat_fur",
+                "scarf_fur",
+                "quiver",
+                "sheath",
+                "shortbow",
+                "arrow_wood",
+                "arrow_wood",
+                "copper_knife",
+                "copper_ax",
+                "pot",
+                "waterskin",
+                "fur_rollmat",
+                "flint_steel",
+                "tinderbox"
```

Comment:
>As for the professions...should I change the Nomad

For mainline either seriously nerf or remove.  As a mod, can be left; cf. Rivet's Roguelike Classes mod, whose professions are generally preferable to mainline starts IIRC.

I'd suggest losing the rollmat, quiver, and sheath right off the bat; pot downgrades to stone pot or removed, one pack of arrows rather than two and losing either the axe or the knife, for starters.  Skills probably need dropped too.